api/src/main/java/uk/gov/dvsa/mot/vehicle/read/core/VehicleReadService.java

```diff
@@ -14,6 +15,8 @@
 
     List<Vehicle> findByRegistration(String registration);
 
+    List<DvlaVehicle> findDvlaVehicleByRegistration(String registration);
```

Comment:
Yea true, but we didn't want to have 40 different dvla models similar to the vehicle models